models/action_recognition/model_templates/custom-recognition/s3d-rgb-mobilenet-v3/template.yaml

```diff
@@ -0,0 +1,58 @@
+name: s3d-rgb-mobilenet-v3
+domain: Action Recognition
+problem: Custom Recognition
```

Comment:
Custom Action Recognition will be clearer

Reply:
done